js/controllers/grid/AuthorReviewRevisionsGridHandler.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @file js/controllers/grid/AuthorReviewRevisionsGridHandler.js
+ *
+ * Copyright (c) 2014-2019 Simon Fraser University
+ * Copyright (c) 2000-2019 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class AuthorReviewRevisionsGridHandler
+ * @ingroup js_controllers_grid
+ *
+ * @brief Author revisions grid handler.
+ */
+(function($) {
+
+	// Define the namespace.
+	$.pkp.controllers.grid.AuthorReviewRevisionsGridHandler = $.pkp.controllers.grid.AuthorReviewRevisionsGridHandler || {};
```

Comment:
Ok, so the issue with the namespace is because this requires a new namespace set up. So to put this file into `/js/controllers/grid/author/AuthorReviewRevisionsGridHandler.js` you would add a second namespace:

```
$.pkp.controllers.grid.author = {};
$.pkp.controllers.grid.author.AuthorReviewRevisionsGridHandler = {};
```

Reply:
So if I want to have the script in `lib/pkp/js/controllers/grid/files/review/AuthorReviewRevisionsGridHandler.js`

I should have

```
// Define the namespace.
$.pkp.controllers.grid.files.review  = {};
$.pkp.controllers.grid.files.review.AuthorReviewRevisionsGridHandler = {};
```

?

Reply:
The namespace and the file location do not need to match. But if you want something in a name space like `$.pkp.a.b.c.mything`, and `$.pkp.a.b.c` has not been initialized as an empty object, you have to initialize the empty object first.

The file location should only depend on where the file is located and how it is referenced in `minifiedScripts.txt`. I'm happy to take a look at that if it's not working for you. Just get everything else committed and let me know for the next review.

Reply:
got this working from within the correct folder..